test/custom_function_replacement_test.cc

```diff
@@ -0,0 +1,15 @@
+/* ----------------------------------------------------------------------------
+ * SymForce - Copyright 2022, Skydio, Inc.
+ * This source code is under the Apache 2.0 license found in the LICENSE file.
+ * ---------------------------------------------------------------------------- */
+
+#include "symforce_function_codegen_test_data/sympy/custom_function_replacement.h"
+
+#include <math.h>
```

Comment:
Kind of a nit-pick, but `#include <cmath>` instead of `#include <math.h>` is more idiomatic for C++